frame/chainbridge/src/lib.rs

```diff
@@ -618,4 +618,8 @@ impl<T: Trait> EnsureOrigin<T::Origin> for EnsureBridge<T> {
             r => Err(T::Origin::from(r)),
         })
     }
+
```

Comment:
Shouldn't we add this before the method:
```
#[cfg(feature = "runtime-benchmarks")]
```

Reply:
I added in the latest commit.

Reply:
Remove space. It should be the same padding for function and annotation
![Screenshot from 2021-04-07 14-48-26](https://user-images.githubusercontent.com/2946193/113861838-5588d080-97b0-11eb-8131-467455e0e0d6.png)

Reply:
So strange, in my VS code it is same. I will fix its.